router/link/link_events.go

```diff
@@ -121,6 +122,16 @@ func (self *linkDestUpdate) ApplyListenerChanges(registry *linkRegistryImpl, des
 					registry.evaluateLinkState(newLinkState)
 				} else {
 					log.Info("link already known")
+					if existingLinkState.listener.Address != listener.Address {
+						log.Info("link address changed, updating")
```

Comment:
imo, it would be best to see what it went from an to in this log message.

Reply:
update log message. I think the 'link already known message' is consistent. It lets you know this isn't  new link, which is true regardless of whether the link changed or not.

Reply:
sure, i just find it redundant is all :)

Reply:
For links I tend to err on the side of verbosity, since link events tend to be comparatively low in number.